scripts/spcgeonode/docker-compose.yml

```diff
@@ -22,10 +22,25 @@ x-common-django:
     - TIME_ZONE=${TIME_ZONE}
     - ALLOWED_HOSTS=['nginx','127.0.0.1','localhost','$HTTPS_HOST','$HTTP_HOST']
     - SECRET_KEY=${SECRET_KEY}
+
+    ## UNCOMMENT FOR SSL
+    #- GEOSERVER_PUBLIC_LOCATION=https://${HTTPS_HOST}/gs/
+    #- GEOSERVER_WEB_UI_LOCATION=https://${HTTPS_HOST}/geoserver/
+    #- SITEURL=https://${HTTPS_HOST}
+    ## COMMENT FOR SSL
+    - GEOSERVER_PUBLIC_LOCATION=http://${HTTP_HOST}/gs/
+    - GEOSERVER_WEB_UI_LOCATION=http://${HTTP_HOST}/geoserver/
+    - SITEURL=http://${HTTP_HOST}
+
+    # database info, ${variables} are read from .env
+    - DATABASE_URL=postgres://${GEONODE_DATABASE}:${GEONODE_DATABASE_PASSWORD}@postgres:5432/${GEONODE_DATABASE}
+    - GEODATABASE_URL=postgis://${GEONODE_GEODATABASE}:${GEONODE_GEODATABASE_PASSWORD}@postgres:5432/${GEONODE_GEODATABASE}

```

Comment:
So, 3 different users for the same database  with two different password? Is there a way to make it simpler?

Reply:
For me this is ok: one superuser (postgres), one just for django stuff (genode) and one for the geospatial data (geonode_data).

Actually there are two databases involved: `geonode` for the django stuff and `geonode_data` for spatial data. I just took the default geonode docker setup as concept and inserted it into the spc setup. 

You could do it all with just one account, too. Do you think that would be better?

Reply:
@gannebamm what about a third database for geofence rules?

Reply:
(I think in this context it would suffice to have just one user for Django and postgis, just my two cent)

Reply:
@t-book The Geofence rules database is a good idea, but imho not part of this pull request.

Reply:
No problem having different databases, but I am not sure having more than 2 users (superuser + regular user) would have an impact on security, as long as the same application has access to these passwords.

Nothing against this solution by the way, it is just for discuss and understand if there is something we can improve.

Reply:
This would require changes to the dockerfile which currently do not reside in this repository. It should get its own issue with label minor imho